src/main/java/openfasttrack/importer/specobject/ImportHelper.java

```diff
@@ -105,46 +105,51 @@ private void foundStartElement(final StartElement element) throws XMLStreamExcep
         switch (element.getName().getLocalPart())
         {
         case SPECDOCUMENT_ROOT_ELEMENT_NAME:
-            LOG.finest("Found XML root element '" + SPECDOCUMENT_ROOT_ELEMENT_NAME + "' at "
+            LOG.finest(() -> "Found XML root element '" + SPECDOCUMENT_ROOT_ELEMENT_NAME + "' at "
```

Comment:
It is faster like this, because the String is not built when the log level is not active.